lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -478,6 +471,20 @@ private void tryLocks(LockClient client, LockRequest request, BlockingMode block
         }
     }
 
+    private void logAtDebugOrInfo(String lockId, LockClient currentHolder, long duration) {
+        if (log.isDebugEnabled()) {
+            log.debug("Blocked for {} ms to acquire lock {} {}.",
+                    duration,
+                    lockId,
+                    currentHolder == null ? "successfully" : "unsuccessfully");
+        } else {
+            log.info("Blocked for {} ms to acquire lock {} {}.",
```

Comment:
It would be useful to send these logs to a separate appender, and provide docs for how to set up an appender (or how we do it, if we can do it automatically).